omod-1.10/src/test/java/org/openmrs/module/attachments/rest/AttachmentRestController1_10Test.java

```diff
@@ -409,4 +473,80 @@ public void getAttachmentBytes_shouldDownloadFile() throws Exception {
 		Assert.assertEquals(downloadResponse.getHeader("File-Ext"), fileExtension);
 		
 	}
+	
+	@Test
+	public void doSearch_shouldGetAttachmentsWithMimeTypesAndContentFamilySet() throws Exception {
+		
+		// Setup
+		String fileCaption = "Test file caption";
+		String mimeType = "application/octet-stream";
+		String fileExtension = "dat";
+		String fileName = "testFile." + fileExtension;
+		
+		Patient patient = createBasicPatient();
```

Comment:
Even this shouldn't be here. I used a patient with ID 6 and changed this line to 
```java
Patient patient = Context.getPatientService().getPatient(6)
```

@gitcliff did you delete a commit? This PR was at 14 commits yesterday and now I see just 13.